src/genie/libs/parser/asa/show_traffic.py

```diff
@@ -0,0 +1,177 @@
+            'packets_output_1_minute': int,
+            'bytes_output_1_minute': int,
+            'packets_drop_rate_1_minute': int,
+            'packets_input_5_minute': int,
+            'bytes_input_5_minute': int,
+            'packets_output_5_minute': int,
+            'bytes_output_5_minute': int,
+            'packets_drop_rate_5_minute': int
+        },
+    }
+
+# =============================================
+# Parser for 'show traffic'
+# =============================================
+class ShowTraffic(ShowTrafficSchema):
+    """Parser for
+        * show traffic
+    """
+
+    cli_command = 'show traffic'
+
+    def cli(self, output=None):
+        if output is None:
+            # excute command to get output
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        ret_dict = {}
+
+
+        #Single output from 'show traffic' command
+        # GigabitEthernet0/0:
+        # received (in 1618693.940 secs):
+        # 415466249 packets       29074806307 bytes
+        # 1 pkts/sec      17001 bytes/sec
+        # transmitted (in 1618693.940 secs):
+        # 316636 packets  64502596 bytes
+        # 0 pkts/sec      0 bytes/sec
+        # 1 minute input rate 9452 pkts/sec,  661142 bytes/sec
+        # 1 minute output rate 206 pkts/sec,  41887 bytes/sec
+        # 1 minute drop rate, 0 pkts/sec
+        # 5 minute input rate 11309 pkts/sec,  790978 bytes/sec
+        # 5 minute output rate 0 pkts/sec,  0 bytes/sec
+        # 5 minute drop rate, 0 pkts/sec
+        
+    
+
+        # RegEx named capture group name assigned to key, matched text assigned to values
+        # {'capture-group-name': 'capture group value'}
+        p1 = re.compile(r'^(?P<name>\S+):$')
+        p2 = re.compile(r'^(?P<queue>\S+).+?(?P<duration>\d+\.\d+)\s+\S+')
+        p3 = re.compile(r'(?P<packets>\d+)\s+packets\s+(?P<bytes>\d+)\s+\S*')
+        p4 = re.compile(r'(?P<packets_sec>\d+)\s+pkts\/sec\s+(?P<bytes_sec>\d+)\s+bytes\/sec$')
+        p5 = re.compile(r'^1 minute input rate\s+(?P<packets_input_1_minute>\d+)\s+pkts\/sec,\s+'
+            '(?P<bytes_input_1_minute>\d+)\s+bytes\/sec$')
+        p6 = re.compile(r'^1 minute output rate\s+(?P<packets_output_1_minute>\d+)\s+pkts\/sec,\s+'
+            '(?P<bytes_output_1_minute>\d+)\s+bytes\/sec$')
+        p7 = re.compile(r'^1 minute drop rate,\s+(?P<packets_drop_rate_1_minute>\d+)\s+pkts\/sec$')
+        p8 = re.compile(r'^5 minute input rate\s+(?P<packets_input_5_minute>\d+)\s+pkts\/sec,\s+'
+            '(?P<bytes_input_5_minute>\d+)\s+bytes\/sec$')
+        p9 = re.compile(r'^5 minute output rate\s+(?P<packets_output_5_minute>\d+)\s+pkts\/sec,\s+'
+            '(?P<bytes_output_5_minute>\d+)\s+bytes\/sec$')
+        p10 = re.compile(r'^5 minute drop rate,\s+(?P<packets_drop_rate_5_minute>\d+)\s+pkts\/sec$')
+
+
+
+        for line in out.splitlines():
+            line=line.strip()
+            m = p1.match(line)
```

Comment:
Please put an example output line above each matching statement. IE:

```python
# GigabitEthernet0/0:
m = p1.match(line)
    ...
```

Reply:
added thank you